tests/installation/disable_grub_timeout.pm

```diff
@@ -55,7 +55,7 @@ sub run {
     # Select Timeout dropdown box and disable
     send_key 'alt-t';
     wait_still_screen(1);
-    my $timeout = "-1";
+    my $timeout = get_var('KEEP_GRUB_TIMEOUT') ? '15' : '-1';
```

Comment:
If you set the variable you do not need to have this change because the whole module should not be loaded, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/34bff9f34ebbbf78958a444f1a9dccb1cdf1ef03/lib/main_common.pm#L990

Reply:
Then this change is needed when KEEP_GRUB_TIMEOUT is used, otherwise the installation will be stuck on grub with infinite timeout.

Reply:
Sorry, I don't understand. The test module "installation/disable_grub_timeout" is only scheduled if the test variable *KEEP_GRUB_TIMEOUT* is **not** set, right? So any change in the module does not really matter

Reply:
You are right, the module is not schedules with KEEP_GRUB_TIMEOUT, will remote it.